src/components/Select/MultiSelect.scss

```
@@ -0,0 +1,66 @@
+@import '~om';
+
+.MultiSelect {
+  $self: &;
+
+  min-height: 2.5625rem;
+  position: relative;
+  display: flex;
+  width: 100%;
+
+  &__search {
+    @include input-styles();
+    display: flex;
+
+    &--error {
+      border: 1px solid color(code-red);
+    }
+  }
+
+  &__selected-items {
+    display: flex;
+    gap: 0.3rem;
+    max-width: 80%;
+    overflow: scroll;
+  }
+
+  &__selected-item {
+    display: flex;
+    align-items: center;
+    border: 1px solid black;
+    border-radius: 0.6rem;
+    padding: 0.2rem;
+  }
+
+  &__selected-value {
+    cursor: default;
+  }
+
+  &__exit {
+    margin-left: 3px;
```

```suggestion
    margin-left: space(1);
```
Let's stick to `space` values where possible, it makes the UI more consistent.